domains/nico.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Personal Website",
+  "repo": "https://github.com/NicoGaming77/NicoGaming77.github.io",
+  "owner": {
+    "username": "NicoGaming77",
+    "email": "nicodreyer7@icloud.com",
+    "discord": "nicosnico#7674",
```

Comment:
JSON doesn't support trailing commas, so remove the comma after `"discord"` key, I was asking you to put a comma after the owner object key.

Reply:
I'm sorry but I don't really understand what you mean by "after the owner object key", I'm not very familiar with JSON.

Reply:
Wait I think I got it